.rubocop.yml

```diff
@@ -32,3 +38,12 @@ Lint/AmbiguousBlockAssociation:
 
 Naming/MemoizedInstanceVariableName:
   Enabled: false
+
+# There are some which cause test failures. If enabled this should be run without -A and checked.
+Style/RedundantInitialize:
+  Enabled: false
+
+Style/CaseEquality:
+  Exclude:
+    - 'config/initializers/02_proc.rb'
+    
```

Comment:
Has something dodgy happened to this file?  GitHub both shows a final blank line and indicates that there isn't one.

![image](https://user-images.githubusercontent.com/1999241/163973961-80238315-8770-41de-938e-a6eafd784bb1.png)